data/docs/server-side-rendering.mdx

```diff
@@ -5,6 +5,8 @@ description: How to configure server-side rendering.
 
 You can get access to the CSS string by using the `getCssText` function. This function is made available by the `createStitches` function.
 
+_IMPORTANT: When making components, make sure to import `styled` and other functions from `../stitches.config` instead of from `@stitches/react` inside your components. `getCssText` will only output CSS that has been created in this manner._
```

Comment:
A suggestion to keep a consistent UI and tone of voice:

```suggestion
> Note: if you've [configured](/docs/installation#import-and-use-it) Stitches, ensure to import `getCssText` from your config file. 
```

Reply:
I'm not sure whether this fully covers the problem. I think the user is required to do this, it would suprise me if SSR would work if importing from `@stitches/react` without using config, too.

Reply:
If you've not configured Stitches at all, it should work fine by importing directly from npm. Did you run into an issue where it didn't?

Reply:
Ah I didn't know that, but no, I haven't tried without setting up config.

Reply:
Gotcha, yeah, so I think this clarifies it further. It's not perfect I agree, but we'll be making some bigger overall docs improvements!